docs/source/usage/workflows.rst

```diff
@@ -18,3 +18,4 @@ This section contains typical user workflows and best practices.
    workflows/probeParticles
    workflows/tracerParticles
    workflows/particleFilters
+   workflows/synchrotronRadiation
```

Comment:
I think this does not correctly link because here it should be the filename 
see https://picongpu--4827.org.readthedocs.build/en/4827/ for checking

Reply:
done